lib/task-data/tasks/ssh-exec.js

```diff
@@ -5,7 +5,7 @@
 module.exports = {
     friendlyName: 'Ssh Exec',
     injectableName: 'Task.Ssh.Exec',
-    implementsTask: 'Task.Base.Ssh',
+    implementsTask: 'Task.Base.Ssh.Command',
     options: {
         commands: null
```

Comment:
If there is a schema in base task. There should be problem to put a null in default value here.